src/main/java/org/folio/circulation/domain/RequestQueue.java

```diff
@@ -45,6 +45,10 @@ Request getHighestPriorityFulfillableRequest() {
     return fulfillableRequests().get(0);
   }
 
+  boolean queueContainsRequestOfType(RequestType type) {
```

Comment:
As this is an question to a queue, I think we can avoid the need to include that in the same. I would hope the invocation would read something like `queue.containsRequestOfType(HOLD)`

```suggestion
  boolean containsRequestOfType(RequestType type) {
```